### PR TITLE
support SVGView and css file init from local file, made init methods more safety

### PR DESCRIPTION
**Pull request by InnaLMC**
support SVGView and css file init from local file, not only bundle, made init methods more safety to not get crash if there is no SVG or CSS file

### CODE REVIEW

File: Sources/SVGLoader.swift [hunk 1]
```diff
@@ -32,43 +32,78 @@ public final class SVGLoader {
             }
             """)
         }
+        
+        public func buildHtml(svg: String) -> String {
```
Comment (s2mr) [inline Sources/SVGLoader.swift]: I felt strange that this method is in a Style struct.
What do you think of moving this method to an SVG Loader member function?

**Comment by s2mr** [comment]
Thanks for your contributing!
I made one comment.
I want to hear your opinion.

**Comment by InnaLMC** [comment]
Yep, at first I wanted to keep it as SVG Loader method, but html should be built in init method, so we can't call this method in init, because at that moment self is not defined as well. If you try to do it you'll see error.

**Comment by InnaLMC** [comment]
@kzumu I moved html building from Style to separated struct, please check

File: Sources/SVGLoader.swift (hunk 1 above)
Comment (InnaLMC) [reply Sources/SVGLoader.swift]: @kzumu I fixed it, could you check it?

**Comment by s2mr** [comment]
Thanks!
LGTM🎉

**Pull request merged**